changelog/changelog.ddoc

```diff
@@ -24,14 +24,14 @@ $(SMALL released $1, $2)
 $4
 )
 
-NIGHTLY_VERSION=
-$(DIVC version,
-$(P
-$(B $(LARGE $(LINK2 http://nightlies.dlang.org, Download D nightlies)))$(BR)
-$(SMALL $1)
-)
-$4
-)
+SEP_HEADER_TEXT_NONEMPTY=$(BR)$(BIG $(RELATIVE_LINK2 bugfix-list, List of all bug fixes and enhancements in D $(VER).))
```

Comment:
Some comments describing what these macros do and where they're used sure would be nice.